mender/templates/api-gateway/deployment.yaml

```diff
@@ -94,6 +94,12 @@ spec:
         resources:
 {{ toYaml .Values.api_gateway.resources | indent 10 }}
 
+        env:
+{{- if .Values.api_gateway.storage_proxy.enabled }}
+          - name: MENDER_STORAGE_PROXY
+            value: {{ include "mender.storageProxyUrl" . }}
+{{- end }}
```

Comment:
What is this environment variable for?

Reply:
Indeed there it's used in the Traefik [dynamic configuration[(https://github.com/mendersoftware/saas/blob/master/kubernetes/mender_stack/gateway/config-mender-api-gateway-traefik.yaml#L7), but this environment variable does not affect the config here. With helm we can template this extra router directly although I would prefer having this at the ingress level and not inside Traefik.
IOW, I think we can drop this one and rely on the ingress.

Reply:
Dropped, thanks